src/main.ts

```diff
@@ -1,3 +1,4 @@
+import 'hammerjs';
```

Comment:
Is this needed?

Reply:
Yes, it's required for angular material.